docs/config-example.md

```diff
@@ -42,7 +42,8 @@ spec:
     #  - name: clean tmps files
     #    bash: |
     #       rm -fr /tmp/kubekey/*
-    
```

Comment:
I think we can remove this line, and the format will look better this way.

Reply:
Do you mean removing the `skipConfigureOS` example line entirely or just making the comment shorter?